keras/utils/layer_utils_test.py

```diff
@@ -111,7 +110,39 @@ def print_to_file(text, end='\n'):
       reader = open(fpath, 'r')
       lines = reader.readlines()
       reader.close()
-      self.assertEqual(len(lines), 34)
+      check_str = """Model: "model_2"
```

Comment:
To avoid spilling on the left side, use string concatenation like this:

```
reference_str = ('Model: "model_2"'
                 '_________________________________________________________________'
                 ' Layer (type)                Output Shape              Param #   '
                 ...
)
```